extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -243,16 +252,56 @@
   display: none !important;
 }
 
-.i-amphtml-story-new-page-control {
-  /* TODO(enriqe): this is a placehoder, replace when new icon is ready. */
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 48 48" fill="#FFFFFF"><path d="M0 0h48v48H0z" fill="none"/><path d="M22 34h4V22h-4v12zm2-30C12.95 4 4 12.95 4 24s8.95 20 20 20 20-8.95 20-20S35.05 4 24 4zm0 36c-8.82 0-16-7.18-16-16S15.18 8 24 8s16 7.18 16 16-7.18 16-16 16zm-2-22h4v-4h-4v4z"/></svg>') !important;
+.i-amphtml-story-has-new-page-notification-container {
+  display: flex !important;
+  position: absolute !important;
+  justify-content: center !important;
+  z-index: 100002 !important; /* On top of the progress bar. */
+  top: 32px !important;
+  right: 0 !important;
+  left: 0 !important;
+  opacity: 0 !important;
 }
 
-[i-amphtml-story-has-new-page] .i-amphtml-story-new-page-control {
-  display: block !important;
+.i-amphtml-story-has-new-page-text-wrapper {
+  background-color: rgba(32, 33, 37, 0.8) !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  padding-top: 4px !important;
+  padding-bottom: 4px !important;
+  padding-right: 10px !important;
```

Comment:
I imagine the compiler is smart enough to optimize this, but even for readability I think we could do `padding: 4px 10px 4px 0;`

Reply:
Done.